examples/java/build.gradle

```diff
@@ -21,7 +21,7 @@ import groovy.json.JsonOutput
 plugins { id 'org.apache.beam.module' }
 applyJavaNature(
   exportJavadoc: false,
-  enableChecker: false,
+
```

Comment:
Noting that we don't apply spotless to the groovy files but we can and it should clean up these blanks and the indentation inconsistencies etc.